neuralhydrology/modelzoo/transformer.py

```diff
@@ -167,17 +173,32 @@ def __init__(self, embedding_dim, dropout, max_len=5000):
         pe = pe[:,:embedding_dim].unsqueeze(0).transpose(0, 1)
         self.register_buffer('pe', pe)
 
-    def forward(self, x, pos_type):
-        """
+        if position_type.lower() == 'concatenate':
+            self.concatenate = True
+        elif position_type.lower() == 'sum':
+            self.concatenate = False
+        else:
+            raise RuntimeError(f"Unrecognized positional encoding type: {pos_type}")
+
+    def forward(self, x):
+        """Forward pass for positional encoding. Either concatenates or adds the positional encoding to encoder input data.
+
+        Parameters
+        ----------
+        x : torch.Tensor
+            Dimension is sequence length x embedding output dimension. 
+            Data that is to be the input to a transformer encoder after including positional encoding. 
+            Typically this will be output from an embedding layer.
 
         Returns
         -------
-        Finish this. 
+	torch.Tensor  
+            Dimension is sequence length x encoder input dimension. 
```

Comment:
Maybe we can be a bit more explicit on the dimension here? Something like this:
```suggestion
            Dimension is ``[sequence length, batch size, encoder input dimension]``.
            The encoder input dimension is either equal to the embedding output dimension (if ``position_type == sum``)
            or twice the embedding output dimension (if ``position_type == concatenate``).
```